src/app/state/blogpost/blogpost.effects.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+import { map, mergeMap } from 'rxjs/operators';
+import { Post } from 'src/app/blogpost/post';

```

Comment:
Unused import - usually in the IDE ctrl + alt + o webstorm (alt + shift + o vscode) will automatically optimize imports